packages/tupaia-web-server/src/routes/DashboardsRoute.ts

```diff
@@ -4,28 +4,40 @@
  */
 
 import { Request } from 'express';
+import camelcaseKeys from 'camelcase-keys';
 import { Route } from '@tupaia/server-boilerplate';
 
-export type DashboardsRequest = Request<
-  any,
-  any,
-  any,
-  any
->;
+export type DashboardsRequest = Request<any, any, any, any>;
 
 export class DashboardsRoute extends Route<DashboardsRequest> {
   public async buildResponse() {
     const { query, ctx } = this.req;
     const { organisationUnitCode, projectCode } = query;
```

Comment:
We were thinking about removing `organisationUnitCode` in favour of `entityCode`, right? What do you think of doing the web-server side of it now, so that eventually it just slots right into the entity server as and when changes are made? I know it's not strictly part of this ticket, but something to consider I guess?

Reply:
@acdunham The back end needs a bit of a clean up anyway so I reckon if the front end looks good let's not let the back end hold it up